VulkanAbstractionLayer/VulkanContext.cpp

```diff
@@ -276,15 +286,24 @@ namespace VulkanAbstractionLayer
         deviceExtensions.push_back(VK_KHR_SWAPCHAIN_EXTENSION_NAME);
         deviceExtensions.push_back(VK_EXT_DESCRIPTOR_INDEXING_EXTENSION_NAME);
         deviceExtensions.push_back(VK_KHR_MULTIVIEW_EXTENSION_NAME);
+        
+#ifdef __APPLE__
+        deviceExtensions.push_back("VK_KHR_portability_subset");
+#endif
```

Comment:
Is there a constant for that?

Reply:
I guess it just like "VK_LAYER_KHRONOS_validation". Didn't find the Macro.